### ISSUE TITLE
renew discussion of beta-thalassemia HBB/LCRB, MONDO:0013517

### ISSUE THREAD

**Issue by maglott** [monarch-initiative/mondo#4229]
I see from https://github.com/monarch-initiative/mondo/issues/1756 that the decision was to treat MONDO:0013517 equivalent to SCDO:0000251

May I request clarification of the distinction between beta thalassmia and Clinical Phenotype of Beta Thalassemia?

is MONDO:0013517 restricted to a genetic basis in the beta globin cluster and its locus control region?
Are its children limited to a genetic bases in those loci as well?
I don't see any siblings of MONDO:0013517. Are there beta thalassemias that that are not caused by rare alleles in the HBB cluster or its locus control region?

I ask because MedGen had not made a distinction between beta thalasemia and  beta-thalassemia HBB/LCRB, and we are trying to determine if we should really split it out from beta thalassemia.

**Comment by kanems** [comment]
As I read the data from OMIM 613985; there are two phenotypes on this entry: Thalassemia, beta (associated with the HBB locus, which is the preferred concept) and Thalassemia, Hispanic gamma-delta-beta (associated with the LCRB, one allele is documented in OMIM https://www.omim.org/entry/152424#allelicVariants ).
This second phenotype maps to Gamma delta beta thalassemia, UMLS CUI: C4016247 (SNOMED CT 788944005) 
So I am not sure that the HBB/LCRB locus term is a useful distinction, either.
I see MONDO:0011381 dominant beta-thalassemia as a child of the beta-thalassemia concept (but it maps to OMIM 603902, with mutations in HBB, so that should be a child of the HBB/LCRB term if Mondo maintains the distinction). The only other child of the beta thalassemia parent concept is MONDO:0008545, thalassemia, beta+, silent allele, which maps to https://omim.org/entry/187550, suggesting it is also an inherited/genetic form but without a confirmed causative variation in HBB or LCRB.

**Comment by nicolevasilevsky** [comment]
@JadeHotchkiss
@Khuthalamnika
@scdodev
@gmazandu

could I please ask you all for your expert advice and thoughts about the proposed revisions above? Thank you!

**Comment by nicolevasilevsky** [comment]
emailed 2022-02-16

**Comment by nicolevasilevsky** [comment]
@pnrobinson and @ahamosh could you help answer these questions?

**Comment by pnrobinson** [comment]
Can we get an easier question please? 
But I think it would be good for Mondo (and OMIM) to have separate numbers for the various subtypes including at least those that are discussed in the OMIM entry
```
Absence of beta globin causes beta-zero-thalassemia. Reduced amounts of detectable beta globin causes beta-plus-thalassemia. For clinical purposes, beta-thalassemia is divided into thalassemia major (transfusion dependent), thalassemia intermedia (of intermediate severity), and thalassemia minor (asymptomatic, carrier state). 
```
Alpha thalassemia -- analogous comments.
@ahamosh  thoughts?

**Comment by kanems** [comment]
Just a gentle reminder that MedGen is still awaiting a decision from Mondo on this so we can appropriately represent this/these concept(s). Thanks!

**Comment by nicolevasilevsky** [comment]
Hi [kanems](https://github.com/kanems), I will try to get an answer to this asap

**Comment by ahamosh** [comment]
Hi Peter and everyone,
I don't think that we will split the B-thalassemia phenotypes.  There is a major push to lump in ClinGen.  Remarkably the Inborn Error of Metabolism decided to lump all the different phenotypes from each enzyme deficiency in to one entry, even for things like Gaucher disease and early-onset vs. late-onset Pompe disease.  In OMIM, it is a mixed bag depending upon whether the phenotypes or the deficiencies were described first.  If you really want all these split, then you will need to push against the prevailing direction of ClinGen.

**Comment by pnrobinson** [comment]
The bioinformatics for supporting genomic diagnostics will be more powerful if we "split" than if we "lump". We are not dependent on ClinCen since we can make disease subclasses in MONDO. It would be good to coordinate between all of our various data resources though! Could we discuss this, it is too difficult a topic for email or github issues!

**Comment by ahamosh** [comment]
Totally agree.  Would need to meet with the ClinGen PIs.  Can you join a DNAC call?  Melissa usually attends.
Thanks,
Ada


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by sabrinatoro** [comment]
I agree with Peter: other users might want/need the more specific terms. 
I think it is ok to "split" in Mondo, as long as we have a "lumped term" in Mondo. ClinGen would use this "lumped term", and other groups could still use the more specific terms.

**Comment by ahamosh** [comment]
I would ask Courtney who coordinates this group.  I would devote a call to it.
Thanks,
Ada

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nicolevasilevsky** [comment]
@sabrinatoro I am reassigning this to you. There was a meeting that was scheduled and it was canceled. Could you please help move this forward and schedule a meeting with the necessary folks if needed?

Thank you!
